app/lib/admin/actions/send_email.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+
+import '../../account/agent.dart';
+import '../../account/backend.dart';
+import '../../package/backend.dart';
+import '../../publisher/backend.dart';
+import '../../service/email/backend.dart';
+import '../../shared/datastore.dart';
+import '../../shared/email.dart';
+
+import '../models.dart';
+import 'actions.dart';
+
+final sendEmail = AdminAction(
+  name: 'send-email',
+  summary: 'Send email(s) to the specified recipients.',
+  description: '''
+Looks up the specified subject's admin emails and/or uses the provided list of
+emails to send out messages. Uses the provided `email-subject` and `email-body`
+without changing anything in it.
+''',
+  options: {
+    'to-email': 'A comma separated list of email addresses.',
+    'to-subject-admin':
+        'A comma separated list of subjects where the admins will get the emails.',
+    'from': 'The email address to impersonate (`support@pub.dev` by default).',
+    'email-subject': 'The subject of the email message.',
+    'email-body': 'The text content of the email body.',
```

Comment:
```suggestion
    'subject': 'The subject of the email message.',
    'body': 'The text content of the email body.',
```

Reply:
I've found the plain `subject` to be too close to confuse with the moderation subjects, and opted for the longer names. We could keep it short, but in that case I'd suggest to use only single `to` parameter, and resolve the emails and subjects within that. wdyt?

Reply:
Updated the PR with the commits: one for dropping the prefix, the other to merge the `to-*` parameters. PTAL!